dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2010, 2020 IBM Corporation and others.
+# Copyright (c) 2010, 2020, 2024 IBM Corporation and others.
```

Comment:
Change the copyright to show _first_year_, _last_year_ (`2010, 2024`), thus remove `2020, `

Reply:
removed 2020